src/Native/System.Native/pal_networking.cpp

```diff
@@ -213,12 +216,33 @@ extern "C" int32_t IPv4StringToAddress(const uint8_t* address, uint8_t* buffer,
     return ConvertGetAddrInfoAndGetNameInfoErrorsToPal(result);
 }
 
+static void AppendScopeIfNecessary(uint8_t* string, int32_t stringLength, uint32_t scope)
```

Comment:
So, I'm still not clear on what our standard is.  I thought from a conversation yesterday that we're doing strings as ```char*``` but other byte arrays as ```uint8_t*```... is that not the case?